src/redux/selectors.jsx

```diff
@@ -0,0 +1,4 @@
+import { useSelector } from "react-redux";
+
+export const user = useSelector((store) => store.login);
```

Comment:
It is more convinient to write `userSelector = (store) => store.login` and then in component call `const user = useSelector(userSelector)` because there is a convention hooks might be used only inside a component